homeassistant/components/sensor/fail2ban.py

```diff
@@ -26,8 +23,9 @@
 
 DEFAULT_NAME = 'fail2ban'
 DEFAULT_LOG = '/var/log/fail2ban.log'
-SCAN_INTERVAL = timedelta(seconds=120)
```

Comment:
Keep the `SCAN_INTERVAL` constant defined. This will set the default scan interval for the platform.